apache-libraries/src/test/java/com/baeldung/apache/beam/intro/WordCountUnitTest.java

```diff
@@ -9,8 +9,8 @@
 
 public class WordCountUnitTest {
     
-    @Test
-    // @Ignore
+//    @Test
```

Comment:
Is there any reason we want to ignore this test?

Reply:
@kwoyke I have re-run the build including the test and now it works fine.